packages/styles/panel.css

```
@@ -3,23 +3,90 @@
   --panel-background-color: var(--white);
   --panel-heading-color: var(--gray-90);
   --panel-padding: var(--space-small);
+  --panel-header-margin: 0 calc(var(--panel-padding) * -1);
+  --panel-divider-border: var(--gray-40);
```

```suggestion
  --panel-divider-border-color: var(--gray-40);
```

Make it more self documenting.